src/js/components/cycleCount/tableCell/QuantityCell.jsx

```diff
@@ -36,37 +43,59 @@ const QuantityCell = ({
     );
   }
 
+  const {
+    onChangeValidationHandler,
+    onBlurValidationHandler,
+    error,
+    shouldShowError,
+  } = useCellValidation({
+    initialValue,
+    cycleCountId,
+    index,
+    fieldName: cycleCountColumn.QUANTITY_COUNTED,
+  });
+
   const dispatch = useDispatch();
 
   const onChange = (enteredValue) => {
     const parsedValue = enteredValue
       ? (parseInt(enteredValue, 10) || 0)
       : enteredValue;
     setValue(parsedValue);
+    onChangeValidationHandler();
   };
 
   const onBlur = () => {
     dispatch(
       updateFieldValue({
         cycleCountId,
         rowId: id,
-        field: 'quantityCounted',
+        field: cycleCountColumn.QUANTITY_COUNTED,
         value,
       }),
     );
+    onBlurValidationHandler();
   };
 
   return (
     <TableCell className="rt-td rt-td-count-step">
       <InputField
         type="number"
         value={value}
+        showErrorBorder={shouldShowError}
```

Comment:
yeah, imho `showErrorBorder` is enough - if we want to be consistent we should either change `shouldShowError` to `showError` as I suggested above or have the prop `shouldShowErrorBorder` but it sounds even weirder.